docs/ingestion/ingestion-spec.md

```diff
@@ -396,6 +396,46 @@ The `filter` conditionally filters input rows during ingestion. Only rows that p
 ingested. Any of Druid's standard [query filters](../querying/filters.md) can be used. Note that within a
 `transformSpec`, the `transforms` are applied before the `filter`, so the filter can refer to a transform.
 
+### Projections
+
+Projections are pre-aggregated segments that can speed up queries by reducing the number of rows that need to be processed. Use the `projectionsSpec` block to define projections for your data during ingestion or [create them afterwards](../querying/projections.md#after-ingestion).
```

Comment:
```suggestion
Projections are a type of aggregation that Druid computes and stores as part of a segment. The pre-aggregated data reduces the number of rows for the query engine to process. This can speed up queries for query shapes that match a projection. 

Define projections for a new data source in the `projectionsSpec` block during ingestion. To add projections for an existing data source, see [create them afterwards](../querying/projections.md#after-ingestion).
```